### PR TITLE
Added PingCountTimerSponge

### PR DESCRIPTION
**Pull request by BrainStone**
This is a implementation for Sponge ping measurement.

### CODE REVIEW

File: Plan/src/main/java/com/djrapitops/plan/system/tasks/SpongeTaskSystem.java [hunk 1]
```diff
@@ -10,6 +14,20 @@
...
+    public void enable() {
+        super.enable();
+        try {
+            PingCountTimerSponge pingCountTimer = new PingCountTimerSponge();
+            Sponge.getEventManager().registerListeners(plugin, pingCountTimer);
+            long startDelay = TimeAmount.SECOND.ticks() * (long) Settings.PING_SERVER_ENABLE_DELAY.getNumber();
+            RunnableFactory.createNew("PingCountTimer", pingCountTimer)
+                    .runTaskTimer(startDelay, PingCountTimerSponge.PING_INTERVAL);
+        } catch (ExceptionInInitializerError | NoClassDefFoundError ignore) {
+            // Running CraftBukkit
```
Comment (Rsl1122) [inline Plan/src/main/java/com/djrapitops/plan/system/tasks/SpongeTaskSystem.java]: This exception is a leftover from the copied class. The exception there is caused by Reflection using spigot methods, which are unavailable when running CraftBukkit.

This try catch can be removed.

**Comment by Rsl1122** [comment]
Thanks for the work. There wasn't much that needs changing. :)

**Comment by BrainStone** [comment]
Fixed what you wanted fixed

**Comment by Rsl1122** [comment]
Looks good to me, cheers.

**Pull request merged**